content/events/22-23/zeustp.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ZeusTP
+created_at: 04-04-2023
+time: '02-05-2023 17:30'
+end: '02-05-2023 22:30'
+location: 'De Kelder'
+locationlink: $kelder
+# banner: "https://beeld.ugent.be/files/photos/.117063/w940q85_Z2020_060_001.jpg"
+# header_text_background: true
+# tags:
+#   - Verkiezingen
```

Comment:
is het stiekem een voorronde van de zeus verkiezingen?

Reply:
ty